src/mrpro/algorithms/optimizers/pgd.py

```diff
@@ -45,8 +33,44 @@ def pgd(
 ) -> torch.Tensor:
     """Proximal gradient descent algorithm for solving problem min_x f(x) + g(x).
 
-    It relies on the implementation of the proximal map of g.
-
+    f is convex, differentiable, and with L-Lispchitz gradient. 
+    g is convex, non-smooth with computable proximal map.
+    
+    For fixed stepsize t, pgd converges globally when t \in (0, 1/L), where L is the Lipschitz constant of the gradient of f.
+    
+    In applications, f is usually of the form f(x) = 1/2 ||Ax - target||^2, where A is a linear operator.
+    In this case, t \in (0, 1/||A^T A||) for convergence.
+    If no backtracking is used, the fixed stepsize should be given accordingly to the convergence condition.
+    
+    
+    Case example: image reconstruction.
+    Problem formulation: min_x 1/2 ||Fx - target||^2 + ||x||_1,
+    with F being Fast Fourier Tranform, target acquired data \in k-space and x \in image space,
+    f(x) = 1/2 ||Fx - target||^2, g(x) = ||x||_1.
+    In this case, ||F^T F|| = 1.
+    
+    
+    Example of usage:
+    fft = FastFourierOp()
+    l2 = L2NormSquared(target=kspace_data)
+    f = l2 @ fft
+    g = L1Norm()
+    fft_norm = 1.
+    suitalbe_stepsize = 0.85 * 1 / fft_norm
```

Comment:
stepsize

Reply:
```
Example:
    L1 regularized reconstruction::
          fft = ....
```
I think this kind of indentation (and the double colon, ::) makes it show up as a literal block in the docu